<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UwxErrorCode</class>
 <widget class="QDialog" name="UwxErrorCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Error Code Lookup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="selector_Tab">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_Lookup">
      <attribute name="title">
       <string>C&amp;ode lookup</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Type your 4-character error code:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combo_Code">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_List">
      <attribute name="title">
       <string>&amp;Full list</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="list_Codes">
           <property name="editTriggers">
            <set>QAbstractItemView::EditKeyPressed</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Search">
      <attribute name="title">
       <string>S&amp;earch</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="edit_Search">
             <property name="placeholderText">
              <string>Enter part of error description...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_Search">
             <property name="text">
              <string>&amp;Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="list_Search"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="edit_Result">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Error Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Copy">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Copy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="StatusBarLayout"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>selector_Tab</tabstop>
  <tabstop>combo_Code</tabstop>
  <tabstop>list_Codes</tabstop>
  <tabstop>edit_Search</tabstop>
  <tabstop>btn_Search</tabstop>
  <tabstop>list_Search</tabstop>
  <tabstop>edit_Result</tabstop>
  <tabstop>btn_Copy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
